docs/user-guide/cli-using-team-managing-credential-security.md

```diff
@@ -1,15 +1,77 @@
+## Secure credential storage
 
-To store or update values for the secure fields (for example, when you want to change your username and password), issue the `zowe config secure` command. If, for example, you want to update several property values in a long list of properties, press Enter to skip a field.
+With the introduction of team profiles in Zowe CLI V2, the **Secure Credential Store (SCS) Plug-in** is deprecated. The `zowe scs` command group is obsolete.
 
-To secure a specific field, issue `zowe config set --secure <property-path>`. For example, `zowe config set --secure profiles.base.properties.password`. When you issue the command for an option that is already secured, the CLI prompts you to enter a new option value.
+Secure credential encryption is now included with the Zowe CLI core application. When a command using a profile with missing `user` and `password` information is issued, Zowe CLI V2 prompts you to enter the username and password by default. To save secure properties and values in the Secure Credential Store, use the `zowe config` command group.
 
-You can use an editor to define options to the secure array in `zowe.config.json`. Any option that you define to there becomes secure/prompted-for.
+## Configuring secure properties
 
-## Changes to secure credential storage
+Create a configuration file and set its secure properties (such as usernames and passwords):
 
-With the introduction of team profiles in Zowe CLI V2, the **Secure Credential Store (SCS) Plug-in** is deprecated. The `zowe scs` command group is obsolete. Secure credential encryption is now included with the Zowe CLI core application.
+1. Open the Zowe CLI command prompt.
 
-Zowe CLI V2 prompts you to enter the `username` and `password` securely by default. Commands in the `zowe config` command group let you manage security for any option value.
+2. To initialize a **project team** configuration file in the current working directory:
+
+    ```
+    zowe config init
+    ```
+    To initialize a **project user** configuration file in the current working directory:
+    ```
+    zowe config init --user-config
+    ```
+    To initialize a **global team** configuration file in the `ZOWE_CLI_HOME` directory:
+    ```
+    zowe config init --global-config
+    ```
+    To initialize a **global user** configuration file in the `ZOWE_CLI_HOME` directory:
+
+    ```
+    zowe config init --global-config --user-config
+    ```
+    A configuration file is created, if one does not already exist.
+    
+    Additionally, the `profiles.base.properties.user` and `profiles.base.properties.password` fields are added to the base profile `secure` array for that configuration file. This stores the username and password in the Secure Credential Store.
```

Comment:
There may be multiple references to Secure Credential Store (SCS) which can give people the impression that we are still using the SCS plug-in 😋 
Do we want to use other terminology for this location?

Reply:
As discussed, we are adding SCS as a term in the [Zowe Glossary](https://docs.zowe.org/stable/appendix/zowe-glossary) and explaining the difference between the plug-in and the technology.